src/styles/legacy_main.css

```diff
@@ -3961,3 +3961,60 @@ input[type=number]::-webkit-outer-spin-button {
     -webkit-appearance: none;
     margin: 0;
 }
+
```

Comment:
@JerryNyoike  hey you need to use tailwindcss to do all the styling in general we don't grow the `legacy_main.css` file check out how to use tailwind at https://tailwindcss.com/

Reply:
let me know if you need help moving this to tailwind  ...